src/main.cpp

```diff
@@ -1025,6 +1025,14 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
     if (fRequireStandard && !IsStandardTx(tx, reason))
         return state.DoS(0, false, REJECT_NONSTANDARD, reason);
 
+    // Don't relay version 2 transactions until CSV is active, and we can be
+    // sure that such transactions will be mined (unless we're on
+    // -testnet/-regtest).
+    const CChainParams& chainparams = Params();
```

Comment:
Nothing above in this function calls Params(); I guess I didn't check the functions called from here to see if they call Params(), but can we please fix that later when we change those functions?  This has many ACKs now and is holding up 0.12.1.